apps/include/opt.h

```diff
@@ -176,6 +178,8 @@
         {"bugs", OPT_S_BUGS, '-', "Turn on SSL bug compatibility"}, \
         {"no_comp", OPT_S_NO_COMP, '-', "Disable SSL/TLS compression (default)" }, \
         {"comp", OPT_S_COMP, '-', "Use SSL/TLS-level compression" }, \
+        {"no_cert_comp", OPT_S_NO_CERT_COMP, '-', "Disable TLSv1.3 certificate compression" }, \
+        {"cert_comp", OPT_S_CERT_COMP, '-', "Use TLSv1.3 certificate compression (default)" }, \
```

Comment:
I'm wondering whether having it on by default is the right choice. There is some processor cost involved in having this on, at the benefit of having a reduced network usage. A classic trade off. Is "on" the right call for everyone though?

Reply:
Chrome has it on by default. How many people enable ZLIB in OpenSSL? To really take advantage of this, it needs to be built with Brotli, as that's the compression algorithm that Chrome uses/prefers.